FPIS/Views/Login.cs

```diff
@@ -11,6 +13,69 @@ public Login()
             InitializeComponent();
             Theme.FormInstance = this;
             Theme.Set(Themes.LIGHT);
+            btnLogin.Click += LogUserIn;
+        }
+
+        private void LogUserIn(object? sender, EventArgs e)
+        {
+            string employeeId = txtEmpId.Text;
+            string password = txtPassword.Text;
+            LoginAuth authToken = AuthenticateUser(employeeId, password);
+            switch (authToken)
+            {
+                case LoginAuth.NOUSERNAME:
+                    MaterialMessageBox.Show("Kindly provide your Employee ID", "Login Failed");
+                    txtEmpId.Focus();
+                    break;
+                case LoginAuth.NOPASSWORD:
+                    MaterialMessageBox.Show("Kindly provide your password", "Login Failed");
+                    txtPassword.Focus();
+                    break;
+                case LoginAuth.AUTH_FAIL:
+                    MaterialMessageBox.Show("Invalid login credentials", "Login Failed");
+                    // The called function currently only clears the password
+                    ResetFields();
+                    break;
+                case LoginAuth.AUTH_PASS:
+                    MaterialMessageBox.Show("Welcome", "Login Passed");
+                    break;
+            }
+        }
+
+        //TODO: Delete this test code
+        public LoginAuth TestAuthenticateUser(string employeeId, string password) => AuthenticateUser(employeeId, password);
```

Comment:
If this is not needed by others, you can remove it 😄

Reply:
You do realize the unit tests will fail should there be the need to run them. This code is probably not useful because I already run the tests and they work just fine. But perhaps one of you might want to run the test to see how it works. But I'll do away with them anyways